extension/src/Registry.ts

```diff
@@ -207,18 +222,26 @@ export class Registry {
     /**
      * Gets the version-specific metadata for a specific version of a package.
      *
-     * If `version` is omitted or `"latest"`, this returns the latest version.
+     * If `version` is the name of a release channel, this gets the latest version in that channel.
+     * If `version` is omitted, this gets the latest version for the user's selected channel.
+     * @throws VersionMissingError if the given version does not exist.
      */
-    public async getPackageVersionMetadata(name: string, version: string = 'latest') {
+    public async getPackageVersionMetadata(name: string, version?: string) {
```

Comment:
I'm fine with this constructing the package itself, but it should probably be renamed to just `getPackage()` or similar.

Alternatively, it could return `{ manifest, version }`, but since everything currently using the function just constructs a package from it anyways, that doesn't seem very useful.